src/pages/blog/airship-update-december-2020.md

```diff
@@ -0,0 +1,134 @@
+[_blog posts_](https://www.airshipit.org/blog/) regularly, including the Airship 2.0 Blog series. These blog posts
+introduce the changes from Airship 1.0 to Airship 2.0, highlight new features, and detail each component's evolution.
+The recommended reading order for these blogs is listed below. It begins with the lessons learned on the road to Alpha,
+to address the changes made from the original blog series 1-6.
+
+- [*Airship 2.0 is Alpha - Lessons Learned*](https://www.airshipit.org/blog/airship2-is-alpha/)
+- [*Pre-Alpha Airship Blog Series 1 - Evolution Towards 2.0*](
+  https://www.airshipit.org/blog/pre-alpha-airship-blog-series-1-evolution-towards-2.0/)
+- [*Pre-Alpha Airship Blog Series 2 - An Educated Evolution*](
+  https://www.airshipit.org/blog/pre-alpha-airship-blog-series-2-an-educated-evolution/)
+- [*Pre-Alpha Airship Blog Series 3 - Airship 2.0 Architecture High Level*](
+  https://www.airshipit.org/blog/pre-alpha-airship-blog-series-3-airship-2.0-architecture-high-level/)
+- [*Pre-Alpha Airship Blog Series 4 - Shipyard - an Evolution of the Front Door*](
+  https://www.airshipit.org/blog/pre-alpha-airship-blog-series-4-shipyard-an-evolution-of-the-front-door/)
+- [*Pre-Alpha Airship Blog Series 5 - Drydock and Its Relationship to Cluster API*](
+  https://www.airshipit.org/blog/pre-alpha-airship-blog-series-5-drydock-and-its-relationship-to-cluster-api/)
+- [*Pre-Alpha Airship Blog Series 6 - Armada Growing Pains*](
+  https://www.airshipit.org/blog/pre-alpha-airship-blog-series-6-armada-growing-pains/)
+
+<br>
+
+Stay tuned for upcoming blog posts as Airship 2.0 progresses through release!
+
+<br>
+
+## **OPEN INFRASTRUCTURE SUMMIT OCTOBER 2020**
+
+Last month there were several excellent presentations related to Airship at the Open Infrastructure Summit. You can find
+each of these below:
+
+* [Airship 101](https://www.youtube.com/watch?v=Fh-YJDpQ5xE) (Andrew Karandjeff, Drew Walters)
+* [Own your YAML: extending Kustomize via Plugins](https://www.youtube.com/watch?v=Xoh_OpLoVtI) (Matt McEuen)
+* [Tackling security challenges with Airship](https://www.youtube.com/watch?v=9Qww2qHhNmE) (Alexander Hughes)
+* [Machine Learning at Edge Cloud](https://www.youtube.com/watch?v=TJlj8oMONPo) (Prakash Ramchandran, Vivek Hariharan)
+* [Significance of Hardware Classification combined with Host Configuration Operator](
+  https://www.youtube.com/watch?v=S0tJapmYNP4) (Digambar Patil, John Williams, Sirisha Gopigiri)
+* [China Mobile software and hardware integrated portable cloud platform test system](
+  https://www.youtube.com/watch?v=8N5PeW2l7Lc) (Xiaoguang Zhang, Zhiqiang Yu)
+* [hardware automation deployment and test with big scale in ChinaMobile NFV cloud](
+  https://www.youtube.com/watch?v=omKxP-qiKng) (Xiaoguang Zhang, Zhiqiang Yu)
+
+<br>
+
+## **AIRSHIP CONTAINER AS A SERVICE**
+
+Two operators will manage the lifecycle of Virtual Machines and their relationship to the cluster: vNode-Operator (ViNO)
```

Comment:
This is a good description of the two operators. It would be nice to also include a small tidbit on the motivation for creating these operators.

Reply:
done

Reply:
Since the Blog section is fairly consistent each time, I might suggest either spotlighting the Airship Container as a Service and Github Labels sections, or moving both sections above the blog so readers see these newer updates before needing to scroll since they're pretty substantial.